buildpack_plan.go

```diff
@@ -20,8 +20,8 @@ import (
 	"fmt"
 	"reflect"
 
+	"dario.cat/mergo"
```

Comment:
why the change?

Reply:
It came up as needing an update, so I did. Nothing beyond that. I don't think we need to, but we're in a breaking change window, so it seemed like a good time.